data/free worlds reconciliation.txt

```diff
@@ -970,7 +970,7 @@ mission "FW Pug 1 Return"
 	
 	on offer
 		conversation
-			`You receive an urgent message from JJ: "Captain <last>, return to New Tibet at once! We've received word that the Syndicate is somehow severing their hyperspace connections to the rest of human space. We're calling an emergency council meeting, to convene as soon as you arrive here."`
+			`You receive an urgent message from JJ: "Captain <last>, return to New Tibet at once! We've received word that the Syndicate is somehow severing their hyperspace connections to the rest of human space. We're calling an emergency Council meeting, to convene as soon as you arrive here."`
```

Comment:
I think every other change is good aside from this one. Council meeting is a generic phrase and doesn’t need to be capitalized, even if it is referring to a meeting of the Free Worlds Council.

If I just Google council meeting, I get results from local governments; at least here, City Council is capitalized (usually preceded by the city name, or city is swapped for county), but council meeting is not.

Reply:
Yes, "council meeting" is a generic phrase when referring to a meeting of a council. However, in this instance I capitalized it because I read it as "an emergency meeting of the Council".
Likewise, there are Security Council meetings ( https://www.un.org/en/sc/meetings/ ) and Foreign Affairs Committee meetings in national parliaments.

Reply:
Those examples include the name of the council/committee, though. In this case, if JJ had said “We’re calling an emergency Free Worlds Council meeting,” then capitalizing council would make sense, but the full name isn’t used.

Reply:
The full name isn't used, that would be an "United Nations Security Council meeting" or a "Canadian House of Commons Standing Committee on Foreign Affairs and International Development meeting". In practice short names are used, e.g. the "United States Senate" is usually called simply the Senate, and thus there is "Senate seniority", "Senate sessions", or a "Senate vacancy".

Likewise, there are dozens of occurences of the Council (and the Senate) without Free Worlds.